index.html

```diff
@@ -1,5 +1,7 @@
+<!doctype html>
```

Comment:
W3c validator did not complain it was lowercase. 
En route for strata. Should we fight on that over lunch?

Envoyé de mon iPhone

> Le 19 févr. 2015 à 05:49, Kyle Kelley <notifications@github.com> a écrit :
> 
> In index.html:
> 
> > @@ -1,5 +1,7 @@
> > +<!doctype html>
> Is doctype supposed to be in all capitals?
> 
> —
> Reply to this email directly or view it on GitHub.
>